packages/design/src/config/SequenceElementEdit.tsx

```diff
@@ -118,12 +119,12 @@ const SequenceElementEdit: FormElementComponent<SequenceElement> = ({
             <input type="hidden" {...register(`${element.id}.id`)} />
             <input type="hidden" {...register(`${element.id}.type`)} />
             <input type="hidden" {...register(`${element.id}.elements`)} />
-            {elements.map(elements => (
+            {elements.map(el => (
```

Comment:
I was tracking down an error and having a bunch of nested objects using the same variable name was making me slightly bananas. This doesn't strictly need to be changed